snippets/stylesheet.liquid

```diff
@@ -0,0 +1,90 @@
+{% comment%}
+    # loadTemplateStylesheet.
+    # This will load styles for each template files individually.
+{% endcomment %}
+{% assign loadOptimizedTemplateStylesheet = template.name %} 
+{% case loadOptimizedTemplateStylesheet %}
+
+
+{% when 'index' %}
```

Comment:
Sections can be addd on every pages so I supposed the sections will end up unstyled if you add them on something different than home page